src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -17,15 +17,32 @@
 #include "pandaproxy/logger.h"
 #include "pandaproxy/schema_registry/errors.h"
 #include "pandaproxy/schema_registry/sharded_store.h"
+#include "src/v//pandaproxy/schema_registry/protobuf/confluent/meta.pb.h"
```

Comment:
Improving the situation here is captured in this ticket. https://redpandadata.atlassian.net/browse/CORE-6793. On the plus side, these are included on in a single compilation unit, and we've altered the include path only for this specific file in cmake.

Reply:
Shared issue with https://github.com/redpanda-data/redpanda/pull/22714

Reply:
Why is it `src/v//pandaproxy` instead of `src/v/pandaproxy`?

Reply:
It's a typo, I'll fix it. I guess the preprocessor don't care

Reply:
fixed